index.html

```diff
@@ -30,7 +35,7 @@ <h1 class="title blue">Hey there. I'm Victor<br><span class="green"> I'm a softw
         <a class="downArrow" href="#MyRecentWork"><img src="images/arrowButton.png" alt="down arrow"></a>
     </section>
 
-    <section class="tittleContainer myRecentWork">
+    <section class="tittleContainer myRecentWork" id="Work">
```

Comment:
- [ ] Great job working on your projects section! 💯 However, there's a minor issue with the spacing between the title and the project cards. Please fix this in order to stick as much as possible with the Figma design! 🥇 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199805096-56f158d7-c91b-41a1-b926-e0e72a11c5c7.png) | ![image](https://user-images.githubusercontent.com/77069011/199804986-ec81acbc-5936-4ca4-930c-fbb2e870d2a7.png) |

Reply:
- [ ] Great job implementing the required changes. However the mobile version now is different from the Figma design. Please fix this in order to stick as much as possible to the template.

Figma template

<img width="249" alt="Screen Shot 2022-11-03 at 10 06 53 PM" src="https://user-images.githubusercontent.com/47673311/199863756-ba49bee2-65f9-4d32-b81a-98aee73839ba.png">

Your design

<img width="249" alt="Screen Shot 2022-11-03 at 10 07 17 PM" src="https://user-images.githubusercontent.com/47673311/199863759-7264a8af-333a-491f-9e08-c877a97d7b64.png">